.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,91 @@
+name: Bug Report
+description: Use this form to report a bug with Citizens.
+
+body:
+  - type: 'markdown'
+    attributes:
+      value: |-
+        ## DO NOT IGNORE
+        Please fill out all required fields the way they are described as.
+        **Not following the issue form will get it closed automatically!**
+        
+        ----
+        
+        ### Section 1) Version Infos
+        Include the outputs of `/version` and `/version Citizens` in the respective fields.
+  - type: 'textarea'
+    attributes:
+      label: '`/version` output'
+      description: 'Include the **complete** output of the `/version` command here.'
+      placeholder: 'This server is running ...'
+      render: txt
+    validations:
+      required: true
+  - type: 'textarea'
+    attributes:
+      label: '`/version Citizens` output'
+      description: 'Include the **complete** output of the `/version Citizens` command here.'
+      placeholder: 'Citizens Version ...'
+      render: txt
+    validations:
+      required: true
+  - type: 'markdown'
+    attributes:
+      value: |-
+        ----
+        
+        ### Section 2) Log and Configuration files
+        Provide your server's latest log file and the configuration file (`config.yml`) of Citizens.
+        
+        When your problem is a Performance issue will you need to also include the `saves.yml` of Citizens.
+  - type: 'input'
+    attributes:
+      label: 'Log file'
+      description: 'Upload your server''s latest.log on pastebin or similar and share the URL.'
+      placeholder: 'https://pastebin.com/...'
```

Comment:
The link in the old issue template is outdated, should be updated to https://paste.denizenscript.com/New/Log

Also the link should be included in the description text (ie so they it can be clicked)